lib/modules/platform/types.ts

```diff
@@ -242,7 +242,7 @@ export interface Platform {
   ensureIssue(
     issueConfig: EnsureIssueConfig,
   ): Promise<EnsureIssueResult | null>;
-  massageMarkdown(prBody: string): string;
+  massageMarkdown(prBody: string, rebaseLabel?: string): string;
```

Comment:
I don't like this (seems like bad design), but I'm also not sure if there's a better way.

Reply:
@rarkins I'm not sure about that. The tickbox in GitHub for example is way better than having to add a label.

Also, if editing a PR title (to add rebase!) is as simple in other platforms as they are in GitHub, it is probably not worth it.

Maybe it's indeed worth for some platform beyond Gerrit. I'd rather let their "maintainers" know and decide whether it's good or not.

Also, I had tried several approaches to get this done. This was by far the most cleanest.

Would it improve if I add jsdocs to this function mentioning it's only used by Gerrit?